### PR TITLE
6 new GHSA sync'ed advisories

### PR DESCRIPTION
**Pull request by jasnow**
6 new GHSA sync'ed advisories

### CODE REVIEW

File: gems/grpc/CVE-2023-32731.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  When gRPC HTTP2 stack raised a header size exceeded error, it skipped
+  parsing the rest of the HPACK frame. This caused any HPACK table mutations to also
+  be skipped, resulting in a desynchronization of HPACK tables between sender and
+  receiver. If leveraged, say, between a proxy and a backend, this could lead to requests
+  from the proxy being interpreted as containing headers from different proxy clients
+  - leading to an information leak that can be used for privilege escalation or data
+  exfiltration. We recommend upgrading beyond the commit contained in  https://github.com/grpc/grpc/pull/32309
+cvss_v3: 7.4
+patched_versions:
+  - ">= 1.53.0"
```
Comment (postmodern) [inline gems/grpc/CVE-2023-32731.yml:18]: The NVD entry says versions `>= 1.53.0, < 1.55.0` are effected. Not sure where they are getting `1.53.0` from, but I think patched versions should be `>= 1.55.0`.

File: gems/jquery-rails/CVE-2012-6708.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  ## Proof of Concept
+  ```
+  $("#log").html(
+      $("element[attribute='<img src=\"x\" onerror=\"alert(1)\" />']").html()
+  );
+  ```
+
+  ## Recommendation
+
+  Update to version 1.9.0 or later.
```
Comment (postmodern) [inline gems/jquery-rails/CVE-2012-6708.yml:20]: The description text is from the jQuery CVE. The version number should reflect `jquery-rails`, which has different version numbers than upstream jQuery.

File: gems/jquery-rails/CVE-2015-9251.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+url: https://github.com/jquery/jquery/issues/2432
+title: Cross-Site Scripting (XSS) in jquery
+date: 2018-01-22
+description: |
+  Affected versions of `jquery` interpret `text/javascript` responses from cross-origin ajax requests, and automatically execute the contents in `jQuery.globalEval`, even when the ajax request doesn't contain the `dataType` option.
+
+
+  ## Recommendation
+
+  Update to version 3.0.0 or later.
```
Comment (postmodern) [inline gems/jquery-rails/CVE-2015-9251.yml:14]: Same here, text is from the jQuery CVE and mentions a different version from `jquery-rails`.

File: gems/jquery-rails/CVE-2020-7656.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+url: https://snyk.io/vuln/SNYK-JS-JQUERY-569619
+title: Cross-Site Scripting in jquery
+date: 2020-05-20
+description: |
+  Versions of `jquery` prior to 1.9.0 are vulnerable to Cross-Site Scripting. The load method fails to recognize and remove "<script>" HTML tags that contain a whitespace character, i.e: "</script >", which results in the enclosed script logic to be executed. This allows attackers to execute arbitrary JavaScript in a victim's browser.
+
+
+  ## Recommendation
+
+  Upgrade to version 1.9.0 or later.
```
Comment (postmodern) [inline gems/jquery-rails/CVE-2020-7656.yml:14]: and here as well.

File: gems/nokogiri/CVE-2021-3537.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+gem: nokogiri
+cve: 2021-3537
+ghsa: 286v-pcf5-25rc
+url: https://bugzilla.redhat.com/show_bug.cgi?id=1956522
```
Comment (postmodern) [inline gems/nokogiri/CVE-2021-3537.yml:5]: Maybe the primary URL should be `https://github.com/advisories/GHSA-286v-pcf5-25rc` which discusses nokogiri specifically.

File: gems/nokogiri/CVE-2021-3518.yml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+gem: nokogiri
+cve: 2021-3518
+ghsa: v4f8-2847-rwm7
+url: https://bugzilla.redhat.com/show_bug.cgi?id=1954242
```
Comment (postmodern) [inline gems/nokogiri/CVE-2021-3518.yml:5]: Maybe the primary URL should be `https://github.com/advisories/GHSA-v4f8-2847-rwm7` which discusses nokogiri specifically.

**Review by postmodern — CHANGES_REQUESTED** [review]
Noticed some discrepencies between the `description` `Solution`/`Recommendation` instructions and the `patched_versions:`. That will probably confuse end-users.

**Comment by jasnow** [comment]
With all of the issues with my PR, I think a REDO is in order. Thanks for your thoughtful feedback.

File: gems/grpc/CVE-2023-32731.yml (hunk 1 above)
Comment (jasnow) [reply gems/grpc/CVE-2023-32731.yml:18]: redo found that 1.53.1 was the right answer

**Comment by postmodern** [comment]
You can always `git commit --amend` and `git push origin master -f`.

**Pull request closed without merging**